decentraland/helpers/storage.js

```diff
@@ -0,0 +1,59 @@
+import { SecureStore } from "expo";
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+// Storage keys
+const EPHEMERAL_ACCOUNT_PRIV_KEY = "EPHEMERAL_ACCOUNT_PRIV_KEY";
+const MY_ASSETS_LIST = "MY_ASSETS_LIST";
+
+export const storeEphemeralAccount = async account => {
+  const { privateKey } = account;
+  await _storeData(EPHEMERAL_ACCOUNT_PRIV_KEY, privateKey);
+};
+
+export const retrieveEphemeralAccount = async () => {
+  let account = null;
+
+  const privateKey = await _retrieveData(EPHEMERAL_ACCOUNT_PRIV_KEY);
+  if (privateKey != null) account = createFromPrivateKey(privateKey);
+
+  return account;
+};
+
+export const storeMyAssets = async myAssets => {
+  const strMyAssets = JSON.stringify(myAssets);
+  await _storeData(MY_ASSETS_LIST, strMyAssets);
```

Comment:
The next time we have to persist some 3rd thing, it may be worth considering automatically persisting and rehydrating our redux store. But I think that would be premature-abstraction (which I can sometimes be guilty of) to include in this PR.

Reply:
Agree, I know that we discussed recently (maybe yesterday hahaha) about middleware, but I think on the next storage demand it'll become necessary. Any definition which path we should go?

Refs: https://github.com/tasitlabs/tasit/pull/228#discussion_r270548313

Reply:
Haha yep. I'm not sure which path we should go yet.

Reply:
We can resolve this if you like.

Reply:
Resolving this.